### DISCUSSION TITLE
Updated USB Power Trigger to support newer cameras.

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#1009] [Announcements]
The older USB trigger events on H7 through H9, no longer work on H10 onward.  The reason they weren't ported was they weren't reliable. But that left a missing feature for new cameras. Finally the USB Power Trigger has be updated https://gopro.github.io/labs/control/usb/ using the more advances scripting support by the new cameras. I have included a demo on how the script was created to you can it enhance further.

**Comment by milsky99** [comment]
Battling to get this to work on any platform blackly, this link or a range of others, go pro boots up fine, no recording started and does not switch off until the timeout kicks in... no clue where to go with this

**Reply by dnewman-gpsw** [reply]
Share a video I what is happening and what QR Codes you are trying.
